src/corecel/sys/ScopedProfiling.cc

```diff
@@ -26,34 +26,44 @@ namespace celeritas
 /*!
  * Whether profiling is enabled.
  *
- * This is true only if the \c CELER_ENABLE_PROFILING environment variable is
- * set to a non-empty value. Profiling is never enabled if CUDA/HIP/Perfetto
- * isn't available.
+ * This is true by default when compiling with Perfetto enabled. The
```

Comment:
> This is true by default ...

humm ... pondering if this is the most intuitive choice.

Reply:
> that they're looking for profiling results "most of" the time. 

That's a good point.  Similarly the `-fprofile` compiler flags leads to always on profiling.  But then conversely why/when would you want to disable it?

Reply:
So maybe the right comparison is the way the GPU profiling works (on only if run under nsys) in which case it would seem the default should be off. ie. profiling on only if:
```
nsys myexec
CELER_ENABLE_PROFILING=true myexec
```

Reply:
> Disabling is useful to measure the overhead of the profiling calls, I suppose?

Wouldn't the perfecto-disabled build be a better yardstick?